config.nci.json

```diff
@@ -30,7 +30,7 @@
     "submit_wps_component" : "false",
 
     # shell environment variables to be used for substitutions in this script
-    "environment_variables_for_substitutions":"HOME,USER,PROJECT,TMPDIR",
```

Comment:
I suggest removing the variables USER, PROJECT and TEMPDIR from this entry. The variable PROJECT is not found in the environment variables of my system. And in my opinion, all three variables are not needed in the further process.

Reply:
Ok. Users can always add extras if/when they need them